src/main/resources/init_db.sql

```diff
@@ -0,0 +1,6 @@
+DROP TABLE IF EXISTS books;
+CREATE TABLE `books` (
+    `id` BIGINT NOT NULL PRIMARY KEY AUTO_INCREMENT,
+    `title` VARCHAR(256) NOT NULL,
+    `price` FLOAT NOT NULL
+);
```

Comment:
keep an empty line at the end of each file to avoid this
![image](https://github.com/mate-academy/jv-jdbc-intro/assets/49767049/08289067-ed08-4fdc-9e07-a14fe0586249)